### ISSUE TITLE
Disable Create Account Option

### ISSUE THREAD

**Issue by maramazza** [onetimesecret/onetimesecret#170]
Hi
Is there a way to disable the create account option?

**Comment by delano** [comment]
There's no configuration setting for that at the moment. I'm gonna go ahead and suggest a similar solution, then provide way more detail that you asked for. 

### Partial solution

If you're running your own install, you can modify the header template to remove the authentication links:
https://github.com/onetimesecret/onetimesecret/blob/2e961717/templates/web/header.mustache#L147-L151

```mustache
      {{#authenticated}}
        <a href="/" class="home">{{cust.custid}}</a> {{#colonel}}<a href="{{baseotsuri}}/colonel/" title="" class="nounderline">*</a>{{/colonel}} |
        <a href="{{baseotsuri}}/account" title="Your Account">{{i18n.COMMON.header_dashboard}}</a> |
        <a href="{{baseotsuri}}/logout" title="Log out of One-Time Secret">{{i18n.COMMON.header_logout}}</a>
      {{/authenticated}}
```

This doesn't disable the authentication endpoints though so people can guess or look at code here to figure it out. There's no configuration setting for that yet either, but you can remove all of the plan types other than `:anonymous`:
https://github.com/onetimesecret/onetimesecret/blob/2e961717/lib/onetime.rb#L260-L277

### Additional info 

**About the plans**
- `:anonymous` is the default plan type without authentication
- `:individual_v1` is the one used when someone signs up for a free account.
- All the rest are unnecessary unless you allow for paid accounts. 

**Modifying plans**
[Source](https://github.com/onetimesecret/onetimesecret/blob/2e961717ffe9471b7e2907ac9c38124fdc7a9576/lib/onetime.rb#L280-L292)

Plans are modified by updating the arguments send to `add_plan`. These are the arguments:
```ruby
    attr_reader :planid, :price, :discount, :options
    def initialize planid, price, discount, options={}
      @planid, @price, @discount, @options = self.class.normalize(planid), price, discount, options
    end
```

- `:planid` a symbol, the unique slug
- `:price` and `:discount` a positive real number and a value between 0 and 1. The discount applied to the price becomes the actual price charged
- `:ttl` an integer of seconds, defines the default lifespan for secrets
- `:size` the maximum length of the text content in KB
- `:api` a boolean, whether the REST API can be used
- `:private` a boolean, whether to show the plan on the pricing page.

**Comment by delano** [comment]
This functionality is in `develop` branch and will be included in the upcoming v0.15.0 release in the next few weeks.
